README.md

```diff
@@ -178,7 +166,7 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-If you like this project...
+If you like this project kindly consider giving it a star⭐
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
**[OPTIONAL]**

- [ ] Great work so far but it would be nice if you fix the MIT and Issues link in your readme file to make your readme more descriptive and professional :+1: